services/core-web/src/components/mine/Permit/PermitConditions.tsx

```diff
@@ -442,8 +443,168 @@ const PermitConditions: FC<PermitConditionProps> = ({
   );
 
   const canViewPdfSplitScreen =
-    viewPdf && pdfSplitViewEnabled && permitExtraction?.permit_amendment_document_guid;
-  const hasConditions = latestAmendment?.conditions?.length > 0;
+    viewPdf && pdfSplitViewEnabled && (permitExtraction?.permit_amendment_document_guid || permitAmendmentDocumentGuid);
+
+  const hasConditions = currentAmendment?.conditions?.length > 0;
+
+  useEffect(() => {
+    if (selectedConditionId && !showLoading && currentAmendment?.conditions?.length > 0) {
+      scrollToCondition(selectedConditionId);
+    }
+  }, [selectedConditionId, showLoading, currentAmendment?.conditions]);
+
+  const isViewingLatestAmendment =
+    latestAmendment?.permit_amendment_guid === currentAmendment?.permit_amendment_guid;
+
+  if (!forceViewConditions) {
+    const extractionState = renderExtractionState(
+      isViewingLatestAmendment,
+      isExtractionInProgress,
+      isExtractionComplete,
+      canStartExtraction,
+      permitExtraction
+    );
+    if (extractionState) {
+      return extractionState;
+    }
+  }
+
+  const renderContent = () => (
+    <Row align="middle" justify="space-between" gutter={[10, 16]}>
+      <LatestAmendmentWarning
+        latestAmendment={latestAmendment}
+        currentAmendment={currentAmendment}
+        onViewAmendment={viewPermitAmendment}
+      />
+      <Col span={24}>
+        <Title className="margin-none" level={2}>
+          Permit Conditions
+        </Title>
+      </Col>
+
+      <Col>
+        <Row gutter={10}>
+          <Col>
+            <CoreButton
+              loading={showLoading}
+              type="tertiary"
+              className="fa-icon-container"
+              icon={
+                <FontAwesomeIcon
+                  icon={isExpanded ? faArrowsToLine : faArrowsFromLine}
+                />
+              }
+              onClick={() => setIsExpanded(!isExpanded)}
+            >
+              {isExpanded ? "Collapse" : "Expand"} All Conditions
+            </CoreButton>
+          </Col>
+          <Col>
+            <CoreButton
+              type="tertiary"
+              icon={<FileOutlined />}
+              loading={showLoading}
+              onClick={toggleViewPdf}
+            >
+              {viewPdf ? "Close" : "Open Permit in"} Document Viewer
+            </CoreButton>
+          </Col>
+        </Row>
+      </Col>
+      <Col span={24}>
+        <div className="core-page-content">
+          <Row gutter={[16, 16]}>
+            {formattedPermitConditionCategories.map((category, idx) => {
+              return (
+                <React.Fragment key={category.href}>
```

Comment:
Since the fragment is the child of the row, is the row even recognizing the Col?  It looks like it wouldn't.

Reply:
I guess that's not new code, but 🤷🏼

Reply:
It does indeed. It's just a way of grouping multiple children without adding extra dom elements (there's multiple `Col`s in that statement)

![image](https://github.com/user-attachments/assets/a56a8420-9d99-4c73-9348-b340dba4d3d0)